agent-packaging/packaging-scripts/build_packages_deb.sh

```diff
@@ -15,22 +15,15 @@
 
 set -e
 
-function exit_error
-{
-  echo "build failed"
-  exit 1
-}
+echo "started build..."
 
-trap exit_error ERR
+apt-get -y update && apt-get -y upgrade
+apt-get install -y git-core
+git clone --branch agent-build "https://github.com/GoogleCloudPlatform/osconfig.git"
```

Comment:
why is the branch specified? i think you need a way to get this info from the build, if we're going to use this in presubmits

Reply:
ah... this should have been removed. it was just to test my changes.. will remove it